keywords for demons and diseases

eu-genia opened the pull request:
https://github.com/BetaMasaheft/Documentation/issues/2603#issuecomment-2324684772

eu-genia committed:
keywords for demons and diseases

https://github.com/BetaMasaheft/Documentation/issues/2603#issuecomment-2324684772

nafisa-valieva commented on DemonsDiseases/Barya.xml:
```diff
@@ -0,0 +1,51 @@
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A cause of disease,  frequently target of <term key="Magic">magic prayers</term>; see also <persName ref="ETH1219baryas"></persName>.</p>
```
not <p><term key="demon">Demonic entities</term>? there is a social group, the 'slaves' and there is a deamon, no? @DenisNosnitsin1970 @karljonaskarlsson

CarstenHoffmannMarburg replied:
I would suggest changing it to 'A cause of disease or a demon, often ...'. As I understand it, the Bāryā is often personalised in the form of a demon, as can be seen from its attributes such as Bāryā ṣallim, i.e. "black Bāryā". This could be included in the term "cause", but I think it is useful to be more specific and define it as a demon in this case. The problem that the term has a historical development, as is the case with many magic names and terms, can be dealt with later, after consulting the relevant literature.

karljonaskarlsson replied:
I think Nafisa's observation is correct: ETH1219baryas seems to refer to an ethnic group and not to a type of demons. It is used for a group of military units in Amda Seyon's chronicle, for example

eu-genia replied:
ETH1219baryas  is a social group, the ID is not used for the military unit, see now also https://github.com/BetaMasaheft/Persons/pull/914

nafisa-valieva reviewed (commented):
May be it can be good to add also Ethiopic terms for all English terms you created?

eu-genia commented:
> May be it can be good to add also Ethiopic terms for all English terms you created?

At the moment what I created are keywords. If we want expanded records with more names, bibliographies etc, it may be then necessary to create Authority files as we do e.g. for Art themes, I am not sure

CarstenHoffmannMarburg commented on DemonsDiseases/Aqweyasat.xml:
```diff
@@ -0,0 +1,51 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>ʾaqʷǝyāṣāt</title>
```
If we keep ʾaqʷǝyāṣāt, we should write ʾAqʷǝyāṣāt with a capital first letter as with the other keywords.

CarstenHoffmannMarburg commented on DemonsDiseases/Aqweyasat.xml:
```diff
@@ -0,0 +1,51 @@
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><term key="demon">Demonic entities</term> cause of disease -  frequently target of <term key="Magic">magic prayers</term>. </p>
```
ʾaqʷǝyāṣāt is Plural from qʷǝyṣ "shinbone". I think the idea is, that persons or mule on risky pathways should not be harmed by getting struck by their shinbones, e.g. shall not have accidents on their ways. I think it is more a metaphor for going safe and sound and not so much a demonic entity, magic agent or something, that justifies the creation of a keyword for it.

CarstenHoffmannMarburg commented on DemonsDiseases/Buda.xml:
```diff
@@ -0,0 +1,51 @@
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>As a  cause of disease -  frequently target of <term key="Magic">magic prayers</term>. Similar to <term key="EvilEye"></term>; see also <persName ref="ETH1310Buda"></persName></p>
```
I suggest: "As a cause of disease or misfortune - ..."

CarstenHoffmannMarburg commented on DemonsDiseases/Bleeding.xml:
```diff
@@ -0,0 +1,51 @@
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>As a disease -  frequently target of <term key="Magic">magic prayers</term>. </p>
```
I think these prayers are all about women's menstruation. So we can say: "Female menstruation and menstrual cramps perceived as a disease - ..."

CarstenHoffmannMarburg commented on DemonsDiseases/ChestPain.xml:
```diff
@@ -0,0 +1,51 @@
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>As a disease -  frequently target of <term key="Magic">magic prayers</term>. </p>
```
I have not noticed that it is used that often, but I have no problem with a new keyword.

CarstenHoffmannMarburg commented on DemonsDiseases/Colic.xml:
```diff
@@ -0,0 +1,51 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Colic</title>
```
Either 'Colic' or 'Qʷǝrṣat' - depending on whether we prefer Ethiopic or English terms.

CarstenHoffmannMarburg commented on DemonsDiseases/Dabbas.xml:
```diff
@@ -0,0 +1,51 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>dabbās</title>
```
If we want to capitalise demons as they are persons, we should capitalise Dabbās, as we did with Budā (even if this is not necessarily a proper demon) and Bāryā.

CarstenHoffmannMarburg commented on DemonsDiseases/Dabbas.xml:
```diff
@@ -0,0 +1,51 @@
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A <term key="demon">demonic entity</term> cause of disease -  frequently target of <term key="Magic">magic prayers</term>. </p>
```
I never heard about this demon. The name is perhaps related to dǝbbus "club, iron mace, goad", but I did find the word "dabbās" in Leslau or in [Dillmann](https://betamasaheft.eu/Dillmann/?q=%E1%8B%B0%E1%89%A3%E1%88%B5&mode=none).

CarstenHoffmannMarburg replied:
I found a definition of dabbās in Strelcyn 1972 (Wellcome): "a kind of genius supposed to inhabit places where there are beer and hydromel ... when there is not hydromel, one believes that this genius starts to be malicious and causes illness."

CarstenHoffmannMarburg commented on DemonsDiseases/Dask.xml:
```diff
@@ -0,0 +1,51 @@
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A <term key="demon">demon</term> cause of disease -  frequently target of <term key="Magic">magic prayers</term>. </p>
```
I find, it is useful to list also variant forms like Dasak. But this can also be done more consistently in a later revision.

CarstenHoffmannMarburg commented on DemonsDiseases/DrowningDemons.xml:
```diff
@@ -0,0 +1,51 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Drowning Demons</title>
```
I would prefer "masṭǝma gānen"/"Masṭǝma gānen".

CarstenHoffmannMarburg commented on DemonsDiseases/Falasa.xml:
```diff
@@ -0,0 +1,51 @@
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>As a  cause of disease -  frequently target of <term key="Magic">magic prayers</term>. See also <persName ref="ETH1274Betaes"></persName></p>
```
"As a cause of disease or misfortune - ..."

CarstenHoffmannMarburg commented on DemonsDiseases/Fever.xml:
```diff
@@ -0,0 +1,51 @@
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Fever of undefined origin,  frequently target of <term key="Magic">magic prayers</term>.</p>
```
I think there should be a distinction between ferā (everything that is contagious) and nǝdad (everything that goes with higher temperature). I think in this case we are talking about nǝdad. Ferā might be another keyword, although I think it is less frequent.

CarstenHoffmannMarburg commented on DemonsDiseases/Gudale.xml:
```diff
@@ -0,0 +1,51 @@
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A <term key="demon">demon</term> cause of disease -  frequently target of <term key="Magic">magic prayers</term>. </p>
```
A second form is gʷǝdāle. It may be related to gadala 'to kill', but it is probably better to refrain from a literal explanation for now.

CarstenHoffmannMarburg commented on DemonsDiseases/Malaria.xml:
```diff
@@ -0,0 +1,51 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Malaria</title>
```
If we accept that ferā is always or often malaria, we can use this term. I wonder if it is possible to identify diseases accurately in ancient texts.

CarstenHoffmannMarburg commented:
I am wondering, why I see the message 'All checks have failed' shortly after I have given my review?

<img width="521" alt="Screenshot 2024_All checks have failed" src="https://github.com/user-attachments/assets/4a9f5132-e0c1-4b12-8050-ab09937b617d">

eu-genia commented:
no need to wonder, it is because the files do not validate as long as the schema has not been updated to include the new keywords already used in the records
if you click on the error report behind the link (the one preceded by the red cross) you can always see what does not validate

karljonaskarlsson commented on DemonsDiseases/Buda.xml:
```diff
@@ -0,0 +1,51 @@
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>As a  cause of disease -  frequently target of <term key="Magic">magic prayers</term>. Similar to <term key="EvilEye"></term>; see also <persName ref="ETH1310Buda"></persName></p>
```
Is it a principle to encode types of demons as ethnic groups?

eu-genia replied:
ETH1310Buda is not a demon, it is a term used traditionly for the social group, just like Barya

CarstenHoffmannMarburg commented:
> At the moment what I created are keywords. If we want expanded records with more names, bibliographies etc, it may be then necessary to create Authority files as we do e.g. for Art themes, I am not sure

In my opinion, more information could be added now or later. This would be very useful as many names or terms are ambiguous and need explanation. This can be done in the summary of the files you have created for the keywords. But it might be more visible and practical if each of them, or at least some of them, also got authority files. I think this is more practical because demons, diseases, etc. can also appear in other texts (e.g. chronologies). It could be easily marked up with ref type="authFile", but also with PRS IDs for demons or wizards.

eu-genia commented:
On looking at the list of suggestions https://github.com/BetaMasaheft/Documentation/issues/2627 I think we must for now try to keep the list of keywords to a sensible and manageable minimum, or noone will be able to find anything.

I will give it some more thoughts

eu-genia commented:
(we cannot create PRS IDs for anything other than persons or groups of persons; demons are not persons)

CarstenHoffmannMarburg commented:
> (we cannot create PRS IDs for anything other than persons or groups of persons; demons are not persons)

Except for the Devil himself (PRS11330Devil) and Angels (like PRS13936Kirubel).

eu-genia commented:
> > (we cannot create PRS IDs for anything other than persons or groups of persons; demons are not persons)
> 
> Except for the Devil himself (PRS11330Devil) and Angels (like PRS13936Kirubel).

Devil=Lucifer=Satan as THE personification is OK, but PRS13936Kirubel is not a person but an angel category and should be removed from persons, it was created by @GueshSol who probably was not aware of the rule.

CarstenHoffmannMarburg commented:
> Devil=Lucifer=Satan as THE personification is OK, but PRS13936Kirubel is not a person but an angel category and should be removed from persons, it was created by @GueshSol who probably was not aware of the rule.

We also have Raphael/Rufaʾel in PRS8083Raphael.

CarstenHoffmannMarburg commented:
> > Devil=Lucifer=Satan as THE personification is OK, but PRS13936Kirubel is not a person but an angel category and should be removed from persons, it was created by @GueshSol who probably was not aware of the rule.
> 
> We also have Raphael/Rufaʾel in PRS8083Raphael.

PRS7840Phanuel
PRS7049Michael
PRS9684Uriel
PRS4377Gabriel

CarstenHoffmannMarburg commented:
> > > Devil=Lucifer=Satan as THE personification is OK, but PRS13936Kirubel is not a person but an angel category and should be removed from persons, it was created by @GueshSol who probably was not aware of the rule.
> > 
> > 
> > We also have Raphael/Rufaʾel in PRS8083Raphael.
> 
> PRS7840Phanuel PRS7049Michael PRS9684Uriel PRS4377Gabriel

PRS9036Suriel
PRS8065Raguel

eu-genia commented:
Raphael, Suriel, Michael, Gabriel are personified.
Cherubs and seraphs are just angel ranks.

GueshSol commented:
@eu-genia What I created is the name of a person who owns a certain manuscript I am not referring an angel in my context (see the snapshot image from MS Art institute of Chicago for you reference attached here). If the name of the person is Kirubel though the name originally related to an angel, what is wrong if we create an ID for a person named  Kirubel as we have additional person names of Surafel, Mikael, and the like? I don't see any lack of awareness in this regard.
![Kirubel](https://github.com/user-attachments/assets/0f9abf4d-fa03-49bc-adcc-238c7ac3764e)

eu-genia commented:
Thank you! Then all is good, I did not check attestations, only reacted to what was written in the issue! Owners are of course persons!
(for the future - having a PRS cherub is just like having a PRS angel or a PRS spirit - these are not persons but types of entities, should not have PRS records. Archangels with proper names are of course OK).

eu-genia commented:
It is important to remember that the whole taxonomy (ALL keywords, whether those in the record or not, the whole file) is inserted into each XML record when it is expanded to be displayed, so we should try to keep the taxonomy compact (or find another solution for the future). Every added line increases the size of each and every XML record in the system. While each line is not a lot, when you reproduce it 100 000 times it does take storage space.

I think it is OK to create separate PRS records for "personified" demons (such as Werzelya) etc.

We already have `demon `among the keywords, that can always be used together with the generic `Magic `/ `MagicText `/ `MagicFormula `keywords, and we also have `Asmat` (we can specify there that it also covers _qālāt_).

I think we could add the following to the taxonomy

Magic and spells
- Subtypes (on top of existing MagicText MagicFormula Asmat)
- - --MagicDrawing (for all sorts not covered yet)
- - --MagicSymbols (for magical letters and such)

- Targets of spells 1: Diseases

- - --Disease (any disease)
- - --DiseaseFever (any disease with fever, incl malaria)
- - --DiseasePain (any disease with pain, colic, headache, heartache)
- - --DiseaseMental (any neurological or mental disease, epilepsy/fits, madness)
- - --DiseaseWomen (infertility, menstruation, etc)

- Targets of spells 2: Evil influence, in addition to demon (keyword) Satan (PRS11330Devil) and other personifiable demons

- - --EvilEye
- - --EvilSpirit (generic for all others, if needed, one can create sub-IDs, as we do for text chapters, under `EvilSpirit.xml `and then one could, if necessary, point to EvilSpirit#Barya, EvilSpirit#Buda etc.

_Originally posted by @eu-genia in https://github.com/BetaMasaheft/Documentation/issues/2627#issuecomment-2468093215_



One could also imagine creating Narrative units for some frequent topoi

eu-genia committed:
Update taxonomy.xml

nafisa-valieva reviewed (approved):
Thank you! Looks good for me!

CarstenHoffmannMarburg commented on taxonomy.xml:
```diff
@@ -381,57 +381,59 @@
+                <desc>Demonic entities</desc>                
+                <category xml:id="EvilEye"
+                   corresp="https://betamasaheft.eu/authority-files/EvilEye/main">
+                   <catDesc>Evil Eye</catDesc>
```
Here it is probably useful to add some explanations, especially whether ʾaynat and budā shall be represented by EvilEye or not. To my understanding, both terms may fit to this category, but are quiet ambivalent in their meaning (Cf. https://github.com/BetaMasaheft/Documentation/issues/2592).

CarstenHoffmannMarburg commented on taxonomy.xml:
```diff
@@ -381,57 +381,59 @@
+             </category>
+             <category xml:id="MagicSymbols"
+                corresp="https://betamasaheft.eu/authority-files/MagicSymbols/main">
+                <catDesc>Magic symbols or letters (see also Asmat for ʾAsmāt or Secret
```
Probably it is useful to add also Qālāt as a term, that was used by Strelcyn.

eu-genia replied:
> Probably it is useful to add also Qālāt as a term, that was used by Strelcyn.

see above

> We already have `demon `among the keywords, that can always be used together with the generic `Magic `/ `MagicText `/ `MagicFormula `keywords, and we also have `Asmat` (we can specify there that it also covers _qālāt_).

eu-genia committed:
updating keywords for magic-demons-diseases

https://github.com/BetaMasaheft/Documentation/issues/2627
https://github.com/BetaMasaheft/Documentation/issues/2603

CarstenHoffmannMarburg commented on Magic/Subtypes/MagicSymbols.xml:
```diff
@@ -1,11 +1,11 @@
 schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="Bleeding" xml:lang="en" type="auth">
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="MagicDrawing" xml:lang="en" type="auth">
```
xml:id and name must be identical. So you should refer to "MagicSymbols" here.

CarstenHoffmannMarburg commented on Magic/Targets/Disease.xml:
```diff
@@ -30,9 +30,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                <p>As a disease -  frequently target of <term key="Magic">magic prayers</term>. </p>
-               
-                
+                <p>Any disease,  frequently target of <term key="Magic">magic prayers</term></p>
```
According to this definition, we need to use 'Disease' also in work records, that are termed with 'DiseaseWomen', 'DiseaseFever', 'DiseaseMental' or 'DiseasePain'. Would it not be better to add "that are not specified as ...."?

CarstenHoffmannMarburg commented on Magic/Targets/DiseaseWomen.xml:
```diff
@@ -1,11 +1,11 @@
         <fileDesc>
             <titleStmt>
-                <title>ʾaqʷǝyāṣāt</title>
+                <title>Female / gynaeocological disease</title>
```
gynaecological (delete the first o after ae)
